core/src/main/resources/org/apache/calcite/runtime/CalciteResource.properties

```diff
@@ -207,6 +207,7 @@ ArgumentMustBeNumericLiteralInRange=Argument to function ''{0}'' must be a numer
 ValidationError=Validation Error: {0}
 IllegalLocaleFormat=Locale ''{0}'' in an illegal format
 ArgumentMustNotBeNull=Argument to function ''{0}'' must not be NULL
+AtLeastOneArgumentMustNotBeNull=At least one argument to function ''{0}'' must not be NULL
```

Comment:
there are other functions which give an error in this case, but this error message is frankly better. maybe we should update the other ones.

Reply:
Do you know what I can look for? I could update them.

Reply:
I have seen this: "Illegal use of ''NULL''"

Reply:
let's leave this for another PR, this has been strenuous enough